recipes/h5pp/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+import os, re
+
+class h5ppConan(ConanFile):
+    name = "h5pp"
+    version = "1.8.2"
+    description = "A C++17 wrapper for HDF5 with focus on simplicity"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/DavidAce/h5pp"
+    topics = ("conan","h5pp","hdf5", "binary", "storage")
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    requires = "eigen/3.3.7", "spdlog/1.7.0", "hdf5/1.12.0"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = {
+        'tests'     :[True,False],
+        'examples'  :[True,False],
+        'verbose'   :[True,False],
+        }
+
+    default_options = (
+        'tests=False',
+        'examples=False',
+        'verbose=False',
+    )
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        tools.check_min_cppstd(self, "17")
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["H5PP_ENABLE_TESTS"]         = self.options.tests
+            self._cmake.definitions["H5PP_BUILD_EXAMPLES"]       = self.options.examples
+            self._cmake.definitions["H5PP_PRINT_INFO"]           = self.options.verbose
```

Comment:
```suggestion
            self._cmake.definitions["H5PP_ENABLE_TESTS"]         = False
            self._cmake.definitions["H5PP_BUILD_EXAMPLES"]       = False
            self._cmake.definitions["H5PP_PRINT_INFO"]           = False
```

Reply:
Thank you for the suggestions. I have made the changes in conanfile.py. 

I also updated the version to 1.8.3 with a minor fix. I noticed that the build script in h5pp was needlessly invoking [cmake-conan](https://github.com/conan-io/cmake-conan) which reads dependencies from conanfile.txt instead of conanbuildinfo.cmake from the parent conan process. I don't know if conanbuildinfo.cmake was getting overwritten but it seemed weird. It shouldn't be a problem now in 1.8.3.

This is my first pull request so if I'm doing it wrong please advice.